android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/app/ConfigService.kt

```diff
@@ -66,4 +68,24 @@ interface ConfigService {
   }
 
   fun provideConfigurationSyncPageSize(): String
+
+  /** Provide a list of custom search parameters */
+  fun provideCustomSearchParameters(): List<SearchParameter> {
+    val activeGroupSearchParameter =
+      SearchParameter().apply {
+        url = "http://example.com/SearchParameter/group-active"
+        addBase("Group")
+        name = GROUP_ACTIVE_SEARCH_PARAM
+        code = GROUP_ACTIVE_SEARCH_PARAM
+        type = Enumerations.SearchParamType.TOKEN
+        expression = "Group.active"
+        description = "Search the active field"
+      }
+
+    return listOf(activeGroupSearchParameter)
+  }
+
+  companion object {
+    const val GROUP_ACTIVE_SEARCH_PARAM = "activeGroup"
```

Comment:
per @ellykits' suggestion 
```suggestion
    const val GROUP_ACTIVE_SEARCH_PARAM = "active"
```

Reply:
updated